client/state/activity-log/log/reducer.js

```diff
@@ -29,10 +34,10 @@ export const logItem = ( state = undefined, { type, data, found, query } ) => {
 export const logItems = keyedReducer( 'siteId', logItem, [ DESERIALIZE, SERIALIZE ] );
 logItems.hasCustomPersistence = true;
 
-export const oldestItemTs = keyedReducer( 'siteId', ( state = 0, action ) => {
+export const oldestItemTs = keyedReducer( 'siteId', ( state = Infinity, action ) => {
 	switch ( action.type ) {
 		case ACTIVITY_LOG_UPDATE:
-			return 0 < action.oldestItemTs ? action.oldestItemTs : state;
+			return isNull( action.oldestItemTs ) ? state : Math.min( action.oldestItemTs, state );
```

Comment:
is there a reason to prefer `isNull()` over `null ===`?

not normally a problem other than _we don't need it_ and it introduces unnecessary bloat in the module due to the way we have to import lodash - not much, but some.

(`isNull()` is great when we need a function, though even then we can also pass along `a => a === null` or something like that - but more cleanly maybe is `list.filter( isNull )` etc…)